src-ghc810/Development/IDE/GHC/HieBin.hs

```diff
@@ -0,0 +1,394 @@
+{-
```

Comment:
I’m increasingly starting to wonder if GHC is the right location for the `.hie` files code. We seem to copy paste basically everything since we need to patch it. Hopefully, things will stabilize at some point :disappointed:

Reply:
It's used by haddock, so there isn't much of an alternative.